### DISCUSSION TITLE
Non-english vocabulary

### DISCUSSION THREAD

**Discussion by olevanss** [NVIDIA-NeMo/NeMo#6279] [Q&A] [answered]
Hello community!

Currently I am looking for a way to incorporate set of custom russian words into Nemo so it could be recognized by ASR correctly. Could you please help me to find some ways to do so?
Do I understand correctly that making a [manifest](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/datasets.html) could help me to solve the problem?
Thank you!

**Comment by VahidooX** [comment, marked as answer]
Are you trying to use a Russian ASR model?
If you referring to some sort of word boosting by incorporating, then you can use word boosting feature of Flashlight decoder as described here: [https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/asr/asr_language_modeling.html#word-boosting](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/asr/asr_language_modeling.html#word-boosting)
